### PR TITLE
Update Norwegian translation

### PR DESCRIPTION
**Pull request by zenseii**
Translation of new strings and fixes to some old.

### CODE REVIEW

File: files/lang/nb.po [hunk 1]
```diff
@@ -154,9 +153,8 @@ msgstr "Velg %{name}"
 msgid "Cannot move last troop"
 msgstr "Kan ikke flytte siste styrke i garnisonen"
 
-#, fuzzy
 msgid "Move the %{name}"
-msgstr "Flytt %{name}"
+msgstr "Flytt %{name} "
```
Comment (zenseii) [inline files/lang/nb.po:157]:
```suggestion
msgstr "Flytt %{name}"
```

File: files/lang/nb.po [hunk 2]
```diff
@@ -569,29 +567,25 @@ msgstr "%{spell} gjør %{value} skadepoeng på én levende skapning."
...
-#, fuzzy
 msgid "The %{attacker}' gaze turns the %{target} to stone!"
-msgstr "Medusaenes blikk forsteiner troppen med %{name}!"
+msgstr "Blikket til troppen med %{attacker} forsteiner troppen med %{target}!"
 
-#, fuzzy
 msgid "The %{attacker}' curse falls upon the %{target}!"
-msgstr "Mumienes forbannelse treffer troppen med %{name}!"
+msgstr ""
+"Forbannelsen til troppen med %{attacker} treffer troppen med %{target}!"
```
Comment (zenseii) [inline files/lang/nb.po:578]:
```suggestion
"Forbannelsen til troppen med %{attacker} legger seg over troppen med %{target}!"
```

File: files/lang/nb.po [hunk 3]
```diff
@@ -569,29 +567,25 @@ msgstr "%{spell} gjør %{value} skadepoeng på én levende skapning."
...
 
-#, fuzzy
 msgid "The %{target} are paralyzed by the %{attacker}!"
-msgstr "Troppen med %{name} ble lamslått av kyklopene!"
+msgstr "Troppen med %{target} blir lamslått av troppen med %{attacker}!"
```
Comment (zenseii) [inline files/lang/nb.po:581]:
```suggestion
msgstr "Troppen med %{target} blir lammet av troppen med %{attacker}!"
```

**Comment by ihhub** [comment]
Hi @zenseii , is this PR ready for merge?

File: files/lang/nb.po [hunk 4]
```diff
@@ -7777,9 +7753,9 @@ msgid ""
 "After saving a young boy from a vicious pack of Wolves, you return him to "
 "his father's manor. The grateful nobleman awards you with a Medal of Courage."
 msgstr ""
-"Etter å ha unnsatt en unggutt fra en flokk med blodtørstige ulver, tar du "
-"han med tilbake til herregården til faren hans. Den takknemlige adelsmannen "
-"belønner deg med en medalje av tapperhet."
+"Etter å ha kommet en unggutt til unnsetning fra en flokk med blodtørstige "
+"varger, tar du han med tilbake til faren hans sin herregård. Den takknemlige "
```
Comment (zenseii) [inline files/lang/nb.po:7757]:
```suggestion
"varger, tar du ham med tilbake til faren hans sin herregård. Den takknemlige "
```

**Comment by ihhub** [comment]
@zenseii , thanks a lot for these changes!

**Pull request merged**